docs/content/any/guides/hierarchies/index.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: "Implement Resource Hierarchies"
+title: "Build Resource Hierarchies"
```

Comment:
"Build Resource Hierarchies" sounds like the guide will explain how to build a resource hierarchy instead of how to write an authorization policy when you already have hierarchically related resources.

other ideas:
- "Working with Resource Hierarchies"
- "Authorization for Resource Hierarchies"

Reply:
Yeah was trying to make it match the other modeling guides. Also make it so "how to" could be in front of it. "Authorize Resource Hierarchies" or "Build Authorization for Resource Hierarchies"?

Reply:
> "Build Authorization for Resource Hierarchies"

ship it

Reply:
```suggestion
title: "Build Authorization for Resource Hierarchies"
```